src/main.rs

```diff
@@ -73,6 +74,17 @@ pub fn main() {
 fn main_inner() -> i32 {
     env_logger::init();
 
+    // [workaround]
+    // Currently sccache breaks RLS with obscure error messages.
+    // Until it's actually fixed disable the wrapper completely
+    // in the current process tree.
+    //
+    // See https://github.com/rust-lang/rls/issues/703
+    // and https://github.com/mozilla/sccache/issues/303
+    if env::var(RUSTC_WRAPPER_ENV_VAR) != Ok("sccache".to_owned()) {
+        env::remove_var(RUSTC_WRAPPER_ENV_VAR);
```

Comment:
Is this condition correct? It says only time we want to keep this env-var is when it _is_ `sccache`?

Reading the #703 I'd have expected something like:
```rust
if env::var(RUSTC_WRAPPER_ENV_VAR).map(|v| v.ends_with("sccache")) == Ok(true) {
    env::remove_var(RUSTC_WRAPPER_ENV_VAR);
}
```

Or, if it's the case that RLS does not support RUSTC_WRAPPER why not simply:
```rust
if env::var_os(RUSTC_WRAPPER_ENV_VAR).is_some() {
    env::remove_var(RUSTC_WRAPPER_ENV_VAR);
}
```

Reply:
Oh dear, thanks, of course it should have been `==` and not `!=`. Looking at #703 once again I think that getting rid of the variable altogether is a better idea.